spec/fixtures/openapi/v1_endpoints.yml

```diff
@@ -0,0 +1,127 @@
+openapi: 3.0.1
```

Comment:
Does the example need to be so big? Can it be shrunk to the minimal failing example?

Reply:
You're totally right! It was merely a copy and paste of another fixture file. I have it on my list to cleanup some of the fixtures, this is officially on it now!

Reply:
It might be one I end up using once I get together with test coverage. I started setting up a few base fixtures generated from the `sandbox` app...I need to regroup on what I was doing with that. 🙃